app/main.py

```diff
@@ -1,13 +1,17 @@
+import os
+
 from fastapi import FastAPI, Request
 from fastapi.staticfiles import StaticFiles
 from fastapi.templating import Jinja2Templates
 
-
 app = FastAPI()
 
-app.mount("/static", StaticFiles(directory="static"), name="static")
+app_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Not sure I understand why you need to add that in order to run the tests?

Reply:
The tests folder currently resides outside the app folder, so when running pytest, app.mount looks for the "static" folder in the current working directory - tests. This fix looks for the static folder path always in the same directory as main.py